lbz/authz/collector.py

```diff
@@ -0,0 +1,16 @@
+from lbz.misc import Singleton
+
+
+class AuthCollector(metaclass=Singleton):
+    # TODO: Add uni tests
+    def __init__(self) -> None:
+        self.possible_authz: list = []
```

Comment:
Maybe it could be named `self.possible_permissions`? We should start being consistent in naming to simplify the code and its results.